spec/generators/rspec/install/install_generator_spec.rb

```diff
@@ -27,7 +27,7 @@ def require_rspec_rails
     match(/^require 'rspec\/rails'$/m)
   end
 
-  def use_transactional_fixtures
+  def have_transactional_fixtures
```

Comment:
Hei! I was going to rename this, but I noticed that the point of my first name was to make the two places where it is used read nice. For example,

```
expect(rails_helper).to have_transactional_fixtures
```

vs

```
expect(rails_helper).to transactional_fixtures_enabled?
```

Do you still want me to change this? Thanks!